src/libraries/System.Threading.Tasks.Dataflow/src/PACKAGE.md

```diff
@@ -2,43 +2,143 @@
 
 <!-- A description of the package and where one can find more documentation -->
 
+Provides dataflow components that are collectively referred to as the *TPL Dataflow Library*.
+This dataflow model promotes actor-based programming by providing in-process message passing for coarse-grained dataflow and pipelining tasks.
 
+Note: This package is part of the larger *Task Parallel library* and is designed to help increase the robustness of concurrency-enabled applications.
 
```

Comment:
I'd leave this out.  The TPL nomenclature was useful when it was a standalone experiment, but once it was incorporated into .NET Framework 4.0, having a separate name for it hurts more than it helps, IMHO.